assets/scripts/main.js

```diff
@@ -1,13 +1,14 @@
+---
```

Comment:
Perhaps worth adding a comment in the front matter here to explain why we need it to be run through liquid so it's not accidentally removed in the future?